charts/newrelic-logging/values.yaml

```diff
@@ -327,6 +327,10 @@ daemonSet:
 # Annotations to add to the resulting Pods of the DaemonSet.
 podAnnotations: {}
 
+# If host network should be enabled for fluentbit pods.
+# There are some inputs like UDP which will require this setting to be true as they need to bind to the host network.
+hostNetwork:
```

Comment:
does this default to `false`? Why not setting it explicitely?

Reply:
I had thought about this, was not sure what is our standard way. I see some places we are leaving field empty e.g. https://github.com/newrelic/helm-charts/blob/af780598c5ea67141fb53ec9995e00119671595f/charts/super-agent/values.yaml#L66 and elsewhere we have explicitly setting it as `false`